src/main/java/dev/nolij/zson/Zson.java

```diff
@@ -127,6 +146,85 @@ public static String escape(String string, char escapeQuotes) {
 		return result.toString();
 	}
 
+	@NotNull
+	@Contract("_ -> new")
+	public static Map<String, ZsonValue> obj2Map(@NotNull Object object) {
+		Map<String, ZsonValue> map = Zson.object();
+		for (Field field : object.getClass().getDeclaredFields()) {
+			if(!shouldInclude(field, true)) continue;
+			Comment comment = field.getAnnotation(Comment.class);
+			String commentValue = comment == null ? null : comment.value();
+			try {
+				boolean accessible = field.canAccess(object);
+				if (!accessible) field.setAccessible(true);
+				map.put(field.getName(), new ZsonValue(commentValue, field.get(object)));
+				if (!accessible) field.setAccessible(false);
+			} catch (IllegalAccessException e) {
+				throw new RuntimeException("Failed to get field " + field.getName(), e);
+			}
+		}
+		return map;
+	}
+
+	@NotNull
+	@Contract("_ , _ -> new")
+	public static <T> T map2Obj(@NotNull Map<String, ZsonValue> map, @NotNull Class<T> type) {
+		try {
+			T object = type.getDeclaredConstructor().newInstance();
+			for (Field field : type.getDeclaredFields()) {
+				if(!shouldInclude(field, false)) continue;
+
+				if (!map.containsKey(field.getName())) {
+					throw new IllegalArgumentException("Missing field " + field.getName() + " in map");
+				}
+
+				boolean accessible = field.canAccess(object);
+				if (!accessible) field.setAccessible(true);
+				setField(field, object, map.get(field.getName()).value);
+				if (!accessible) field.setAccessible(false);
+			}
+			return object;
+		} catch (Exception e) {
+			throw new RuntimeException("Failed to create object of type " + type.getSimpleName(), e);
+		}
+	}
+
+	private static boolean shouldInclude(Field field, boolean includeStatic) {
```

Comment:
Please rename `includeStatic` to `forDeserialization` or something along those lines that more clearly communicates why static fields are included or excluded in some contexts.